modules/nw-egress-service-cr.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-egress-traffic-for-vrf-loadbalancer-services.adoc
+
+[id="nw-egress-service-ovn-cr_{context}"]
+= Egress service custom resource
+
+Define the configuration for an egress service in an egress service custom resource. The following YAML describes the fields for the configuration of an egress service:
```

Comment:
Maybe use the object form here?

```suggestion
Define the configuration for an egress service in an `EgressService` custom resource. The following YAML describes the fields for the configuration of an egress service:
```

Reply:
This sentence starts of sounding like it's going to be a procedure, but it seems laid out more like a reference ultimately (also notice no `content-type` set yet in this module, so maybe still in flux).